app/javascript/packs/cms.ts

```diff
@@ -2,6 +2,7 @@ import $ from 'jquery'
 import 'jquery-ui/ui/widgets/droppable'
 import 'jquery-ui/ui/widgets/draggable'
 import 'jquery-ui/ui/widgets/tabs'
+import 'jquery-ui/ui/widgets/tooltip'
```

Comment:
What do we need this `tooltip` for?

Reply:
Yet another leftover from some testing :)